src/components/data/EntitiesContent.js

```diff
@@ -391,6 +399,9 @@ const EntitiesContent = ({
           setDeletingEntities(false)
           setSelectedEntities({})
           setRefreshKey(_.add(1))
+          Ajax().Metrics.captureEvent(Events.workspaceDataDelete, {
+            ...extractWorkspaceDetails(workspace.workspace)
+          })
```

Comment:
I don't think it's necessary to use the spread operator to create a new object.
```suggestion
          Ajax().Metrics.captureEvent(Events.workspaceDataDelete, extractWorkspaceDetails(workspace.workspace))
```

Reply:
fixing!